src/main/java/frc/robot/Robot.java

```diff
@@ -86,7 +87,12 @@ public void configureButtonBindings() {
                 .whileTrue(drive.robotCentricDrive(() -> -driveController.getLeftX(), () -> -driveController.getLeftY(),
                         () -> -driveController.getRightX()));
         copilotController.back().onTrue(intake.safeStateCmd());
-        copilotController.a().onTrue(intake.intakeGamepiece());
+        copilotController.a().onTrue(commandSequences.intake());
+        copilotController.b().onTrue(commandSequences.shooter());
+        copilotController.x().onTrue(intake.spinOut());
+        copilotController.povUp().whileTrue(commandSequences.moveToAmp());
```

Comment:
Why these POV directions? It seems to me like it would make more sense to have speaker be up, since it's the highest, amp be on the side (since it's in the middle), and intake on the bottom because you pick up from the floor

Reply:
The buttons is for how high the arm is since the amp has the arm go all the way up it is povup, the speaker is in the middle of amp and intake so it is right